distribusjon/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/distribusjon/DistribusjonLoeser.kt

```diff
@@ -63,16 +62,13 @@ class DistribusjonLoeser(
             logger.info("Distribuerte eksternt for journalpostId: $journalpostId")
             sikkerLogger.info("Distribuerte eksternt for journalpostId: $journalpostId json: $journalførtJson")
 
-            Event.create(
-                EventName.INNTEKTSMELDING_DISTRIBUERT,
-                behov.forespoerselId!!,
-                mapOf(
-                    Key.JOURNALPOST_ID to journalpostId,
-                    Key.INNTEKTSMELDING_DOKUMENT to inntektsmelding.toJson(Inntektsmelding.serializer()).toJsonNode()
-                )
-            ).also {
-                publishEvent(it)
-            }
+            rapidsConnection.publishEvent(
+                eventName = EventName.INNTEKTSMELDING_DISTRIBUERT,
+                transaksjonId = null,
+                forespoerselId = null,
```

Comment:
behov.forespoerselId bør vel kunne sendes med?

Reply:
Ja, det skal jeg fikse. Jeg gravde ikke dybt nok for å sjekke om forespoerselId var satt. Til mitt forsvar så måtte jeg innom ca. 10 filer for å bekrefte at det faktisk blir satt.